airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/JobNotifier.java

```diff
@@ -102,6 +100,10 @@ public void failJob(final String reason, final Job job) {
   }
 
   protected NotificationClient getNotificationClient(final Notification notification) {
+    return createNotificationClient(notification);
+  }
+
+  public static NotificationClient createNotificationClient(final Notification notification) {
```

Comment:
I made a version of this:
- as static to be used outside of JobNotifier
- while the other is not static so it can be mocked in the JobNotifier tests
Not sure if that's the correct way of doing this...

Reply:
Yep that's a great way to do it!

Reply:
However I would put it in `NotificationClient`